WebRoot/js/zimbraMail/share/view/htmlEditor/ZmHtmlEditor.js

```diff
@@ -820,7 +820,7 @@ function(id, autoFocus) {
 		object_resizing : true,
         font_formats : fonts.join(";"),
         fontsize_formats : AjxMsg.fontSizes || '',
-		convert_urls : true,
+		convert_urls : false,
```

Comment:
Did you check and confirm that this would cause any regression ?
This particular change was actually introduced here in https://jira.corp.synacor.com/browse/ZCS-1278

Reply:
Have checked the working of ZBUG-1278  and it is working as expected from the ticket ZCS-1278
not causing any regression